rules/0180-pure-ftpd_rules.xml

```diff
@@ -18,21 +18,21 @@
     <if_sid>11300</if_sid>
     <match>[INFO] New connection from</match>
     <description>pure-ftpd: New FTP connection.</description>
-    <group>connection_attempt,pci_dss_10.6.1,gdpr_IV_35.7.d,hipaa_164.312.b,nist_800_53_AU.6,</group>
+    <group>connection_attempt,pci_dss_10.6.1,gdpr_IV_35.7.d,hipaa_164.312.b,nist_800_53_AU.6,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
```

Comment:
CC6.8 is related to malware detection. I think it doesn't fit in rules about connection and authentication.